rotkehlchen/types.py

```diff
@@ -910,6 +910,9 @@ class CostBasisMethod(SerializableEnumNameMixin):
     ACB = auto()
 
 
+ANY_BLOCKCHAIN_ADDRESS_BOOK_VALUE: Final = 'NONE'  # value used to mark in the DB that the address entry is valid for any blockchain  # noqa: E501
```

Comment:
Just a suggestion to shorten it. Addressbook can be one word. Also comment.

```suggestion
ANY_BLOCKCHAIN_ADDRESSBOOK_VALUE: Final = 'NONE'  # blockchain value used to mark in the DB that the address entry is valid for any blockchain  # noqa: E501
```